### ISSUE TITLE
Support for older versions of React (v16)?

### ISSUE THREAD

**Issue by brianfinkelstein** [MomenSherif/react-oauth#221]
Hi @MomenSherif!

Firstly, awesome and clean library!

Anyways, I'm working on a React app that was built several years ago and happens to still be running on v16. It used [another library which is now depreciated](https://github.com/anthonyjgrove/react-google-login) so I've happily stumbled upon yours 🤓 as my preferred replacement. 

I've tried simply installing `react-oauth` into the old app but I see some pretty cryptic error messages when the app tries to load in the browser. I can try to grab them and share here tomorrow if you think this might be helpful to debugging.

While I understand and would agree that the most ideal option would be to slowly upgrade everything in it up to modern/supported versions, I'm on a bit of a time crunch and that is not an option in my case. Noticed [this](https://github.com/MomenSherif/react-oauth/blob/master/packages/%40react-oauth/google/package.json#L54) which gave me some glimmer of hope.

**TLDR; Do you know what the minimum version of React this library will support?**

Thanks!

**Comment by MomenSherif** [comment]
Hello @brianfinkelstein ,

It's really hard to support earlier versions of React before hooks,
You will need any version that supports hooks (>=16.8)

**Comment by MomenSherif** [comment]
You can copy paste from the source code the native implementation for the login and add it to you app

**Comment by brianfinkelstein** [comment]
> 

I'll try to get my app up to v16.8 and see how it goes. Thanks!
